staging/nvidia/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
 name: nvidia
-description: Nvidia GPU driver and device plugin for running Nvidia GPU
+description: Nvidia GPU device plugin for running Nvidia GPU
 keywords:
   - gpu
   - nvidia
-version: 0.3.6
+version: 0.3.7
```

Comment:
Shouldn't we bump this version to `0.4.x` as we're removing a significant part of this addon - the driver?

Reply:
Sounds reasonable. Before I do, are there any actual behavior differences in changing more-major parts of the version?

Reply:
I wouldn't know any such differences @jongiddy